### ISSUE TITLE
Installation docs are outdated and don't work

### ISSUE THREAD

**Issue by terion-name** [thumbor/thumbor#1221]
Installation docs are outdated and don't work.

How to install thumbor on Ubuntu 18.04 not with docker? pip worked but service is not created, I can't setup startup, can't find how to set key. No answers in the docs or in internet.

Please, can you write down instructions and update docs?

**Comment by cristiandean** [comment]
What error are you having?

**Comment by terion-name** [comment]
@cristiandean 
- ppa has no bionic release
- pip install does not create system service — I can start it manually with `thumbor` command but can't make it start with system (maybe can be done with supervisord) and no `/etc/thumbor.key` file, etc

just try to install it on ubuntu 18 (in vagrant for example) using instructions in documentation — no luck

**Comment by gingerlime** [comment]
I would plug https://github.com/MinimalCompact/thumbor as an easy way to get thumbor up and running.

**Comment by terion-name** [comment]
@ggilder I was asking about how to doo this without docker

**Comment by gingerlime** [comment]
@terion-name I would still recommend docker, to make things easier. But you can also look inside the project and see how thumbor and its dependencies are installed (within the Dockerfiles), to get an idea.

**Comment by cristiandean** [comment]
@terion-name I recommend using the Supervisor, we use it at globo.com and it works very well.

**Comment by IlCallo** [comment]
I second this issue: `Errore:6 http://ppa.launchpad.net/thumbor/ppa/ubuntu bionic Release 404  Not Found [IP: 91.189.95.83 80]` appears when trying to install Thumbor from `apt-get`

**Comment by IlCallo** [comment]
Seems broken also using `pip` installation, I get
```
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-48RGo4/pycurl/setup.py", line 944, in <module>
        ext = get_extension(sys.argv, split_extension_source=split_extension_source)
      File "/tmp/pip-build-48RGo4/pycurl/setup.py", line 606, in get_extension
        ext_config = ExtensionConfiguration(argv)
      File "/tmp/pip-build-48RGo4/pycurl/setup.py", line 101, in __init__
        self.configure()
      File "/tmp/pip-build-48RGo4/pycurl/setup.py", line 233, in configure_unix
        raise ConfigurationError(msg)
    __main__.ConfigurationError: Could not run curl-config: [Errno 2] No such file or directory
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-48RGo4/pycurl/

```

**Comment by IlCallo** [comment]
I managed to make it work someway.
I had to install all dependencies listed under "Alternative installation" on docs, except `libjasper-dev` which is not present anymore: `sudo apt install ffmpeg libjpeg-dev libpng-dev libtiff-dev libgtk2.0-dev python-numpy python-pycurl webp python-opencv python-dev python-pip`.
I also installed all dependencies listed into the [docker Dockfile](https://github.com/MinimalCompact/thumbor/blob/master/thumbor/Dockerfile), I don't know if I really needed it or not.

Then I cloned this repository in a local folder, entered it and run `sudo python setup.py install`.

Then I just started running `thumbor` and sequentially installing via `pip` all dependencies it couldn't find (`piexif`, `libthumbor`, `futures`, `derpconf`, `Pillow`, `singledispatch`, `backports_abc`).

After this, it worked. Given the dirty installation, I'll probably have some problems later on...
